src/test/scala/beam/agentsim/agents/choice/mode/RangeSpec.scala

```diff
@@ -46,4 +47,7 @@ class RangeSpec extends FlatSpec {
     assert(!Range("[1:10)").has(10))
   }
 
+  "[10:1]" should "throw RuntimeException" in {
```

Comment:
This took me a second to realize - could it be more generic to say something like "lower value that is greater than higher value" should ...